### PR TITLE
fix: [PIE-3488]: filters div scroll issue on screen resize

### PR DESCRIPTION
**Pull request by PratyushGar**
##### Summary:

Added `height:inherit` to parent div for `overflow:scroll` of child filter addition div on screen resize

##### Jira Links:

[https://harness.atlassian.net/browse/PIE-3488](https://harness.atlassian.net/browse/PIE-3488)

##### Screenshots:

https://user-images.githubusercontent.com/100121280/165361175-7656c423-5741-47f9-b443-467890d52e92.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/10-common/components/Filter/FilterCRUD/FilterCRUD.tsx [hunk 1]
```diff
@@ -436,10 +436,17 @@ const FilterCRUDRef = <T extends FilterInterface>(props: FilterCRUDProps<T>, fil
...
-                <FormikForm>
+                <FormikForm className={css.fullHeight}>
                   {filters && filters.length > 0 ? renderFilterList(formik?.submitCount, formik?.errors) : null}
                   {isEditEnabled ? (
-                    <Layout.Vertical spacing="large" padding={{ top: 'xlarge', left: 'large', right: 'large' }}>
+                    <Layout.Vertical
+                      spacing="large"
+                      padding={{ top: 'large', left: 'large', right: 'large' }}
+                      style={{
+                        overflow: 'auto',
```
Comment (AshwinBhatkal-Harness) [inline src/modules/10-common/components/Filter/FilterCRUD/FilterCRUD.tsx:446]: This style will override the one in the scss file right?

Comment (bhavya-sinha) [reply src/modules/10-common/components/Filter/FilterCRUD/FilterCRUD.tsx:446]: Yes, but we don't have class for this. 
Can we move this style to css file @PratyushGar

Comment (PratyushGar) [reply src/modules/10-common/components/Filter/FilterCRUD/FilterCRUD.tsx]: Thanks for the comments, updated the PR.

**Comment by PratyushGar** [comment]
.

**Pull request closed without merging**